scripts/deploy-production.js

```diff
@@ -1,10 +1,18 @@
 const prompts = require("prompts");
 const { exec } = require("child_process");
+const {
+  createNewTag,
+  deleteTag,
+  getCommitMessages,
+  getLatestTag,
+  isOnMainBranch,
+  isWorkingDirectoryClean,
+  pushTags,
+  runLocalDeploy,
+} = require("./deploy-utils");
 
-const localDeployScript =
-  "yarn build:prod && env-cmd -e production yarn deploy:do-not-use && env-cmd -e production ./scripts/email.sh";
-
-const main = async () => {
+/* Deploy by pushing a git tag, to be picked up and pushed to S3 by Evergreen. */
```

Comment:
nit. since this reads as the git tag is pushed to S3
```suggestion
/* Deploy by pushing a git tag, to be picked up and built by Evergreen, and deployed to S3.
```
or even something more verbose

```suggestion
/* Deployment occurs by pushing a git tag, which is then picked up and built by Evergreen, followed by the deployment of the resulting artifacts to S3. */
```